conda/recipes/rmm/meta.yaml

```
@@ -23,7 +23,7 @@ requirements:
     - cmake >=3.14.0
   host:
     - python
-    - librmm {{ version }}=*_{{ GIT_DESCRIBE_NUMBER }}
+    - librmm {{ version }}
```

Idk if we are still interested in this change, but I think something like this should work

```suggestion
    - librmm={{ version }}=*_{{ GIT_DESCRIBE_NUMBER }}
```